src/transformStateWithClones.js

```diff
@@ -6,8 +6,61 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const COMP_ARR = [];
```

Comment:
why are all your variables with uppercase?
uppercase variables must be only for static constants, for all other use camelCase